core/templates/dev/head/components/social_buttons/social_buttons_directive.html

```diff
@@ -1,21 +1,36 @@
 <script type="text/ng-template" id="components/socialButtons">
+<style type="text/css">
```

Comment:
Please look at existing directives (such as [this one](https://github.com/oppia/oppia/blob/develop/core/templates/dev/head/pages/library/activity_tiles_infinity_grid_directive.html)) and carefully follow their structure, indentation and formatting. Consistency of code style matters in this codebase.